### PR TITLE
Addresses #2952 terminal bronchiole part of secondary pulmonary lobule

### PR DESCRIPTION
**Pull request by aleixpuigb**
Addresses #2952 terminal bronchiole part of secondary pulmonary lobule.
@JoshuaFortriede, as some of the information about it was a bit ambiguous (see ticket #2952), I would appreciate if you could confirm that this edits are appropriate? Thank you!

### CODE REVIEW

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -43847,7 +43847,7 @@ property_value: structure_notes "lacks submucosa and cartilage plates; they have
 [Term]
 id: UBERON:0002187
 name: terminal bronchiole
-def: "The last conducting structure of non-respiratory bronchioles; after this point, the airways have alveoli in their walls." [ISBN:0-397-51047-0, MGI:cwg, MP:0002268]
+def: "A bronchiole that is part of a secondary pulmonary lobule and the last conducting structure of non-respiratory bronchioles; after this point, the airways have alveoli in their walls." [ISBN:0-397-51047-0, MGI:cwg, MP:0002268, PMID:16543587]
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:43850]: Given the ambiguity you detail in the linked issue, keeping the original definition (which doesn't seem contested) and using the relationship "overlaps" may be a less controversial solution. Will defer to @JoshuaFortriede or another SME for confirmation.

Comment (JoshuaFortriede) [reply src/ontology/uberon-edit.obo:43850]: [Webb 2006](https://edisciplinas.usp.br/pluginfile.php/5126574/mod_folder/content/0/Thin-section%20CT%20of%20the%20secondary%20pulmonary%20lobule.pdf?forcedownload=1) (paywall article) states:

`Each secondary lobule is supplied by a small bronchiole`

and [Ohno 2021](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8165947/) states:

`The secondary pulmonary lobule is supplied by a lobular bronchiole `

These definitions match my understanding; that the primary bronchiole is the street into the neighborhood (secondary pulmonary lobule), and the terminal bronchioles are the main streets contained within that neighborhood. As such, I believe the new definition works.

I would also suggest adding that the terminal bronchiole is between the primary bronchiole and respiratory bronchiole. Specifically the terminal is distal to the primary, and proximal to the respiratory.

Here is my cheat sheet that I made to keep these airways/structures and space/segments together. The "Type" or RO might not be completely accurate.
image

Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:43850]: Thank you for your input and clarification, @JoshuaFortriede.

> I would also suggest adding that the terminal bronchiole is between the primary bronchiole and respiratory bronchiole. Specifically the terminal is distal to the primary, and proximal to the respiratory.

In a different ticket #2957 I have already added `'terminal bronchiole' SubclassOf ' distally connected to' some 'respiratory bronchiole'`. However, [mice have none or few respiratory bronchioles](https://doi.org/10.1186/s13326-019-0209-1), and therefore 'terminal bronchioles' are connected to alveolar ducts. Therefore I will have to revert that axiom in a different ticket. On the other side, the lobular bronchiole can be found between the primary bronchiole and the respiratory bronchiole. I would like to investigate a bit more before adding that axiom.

image

Comment (bvarner-ebi) [reply src/ontology/uberon-edit.obo:43850]: Consider using singular form in the def:

A bronchiole that is part of a secondary pulmonary lobule and the last conducting structure of **a** non-respiratory **bronchiole**; after this point, the airways have alveoli in their walls.

**Review by bvarner-ebi — APPROVED** [review]

**Pull request merged**